api/v1/openapi.json

```diff
@@ -0,0 +1,1115 @@
+                                                        "$ref": "#/definitions/IPAM"
+                                                }
+                                        },
+                                        "default": {
+                                                "description": "Unexpected error",
+                                                "schema": {
+                                                        "$ref": "#/definitions/Error"
+                                                }
+                                        }
+				}
+			}
+		},
+                "/ipam/{ip}": {
+			"get": {
+				"summary": "Allocate an IP address",
+				"tags": [ "ipam" ],
+				"parameters": [
+					{ "$ref": "#/parameters/ipam-ip" }
+				],
+				"responses": {
+					"200": {
+                                                "description": "Success"
+                                        },
+                                        "default": {
+                                                "description": "Unexpected error",
+                                                "schema": {
+                                                        "$ref": "#/definitions/Error"
+                                                }
+                                        }
+				}
+			},
+			"delete": {
+				"summary": "Release an allocated IP address",
+				"tags": [ "ipam" ],
+				"parameters": [
+					{ "$ref": "#/parameters/ipam-ip" }
+				],
+				"responses": {
+					"200": {
+                                                "description": "Success"
+                                        }
+				}
+			}
+		},
+                "/policy": {
+			"get": {
+				"summary": "Retrieve policy",
+				"tags": [ "policy" ],
+				"parameters": [
+					{ "$ref": "#/parameters/policy-path" }
+				],
+				"responses": {
+					"200": {
+                                                "description": "Success",
+                                                "schema": {
+							"$ref": "#/definitions/PolicyTree"
+                                                }
+					},
+                                        "404": {
+                                                "description": "Policy tree not found",
+                                                "schema": {
+                                                        "$ref": "#/definitions/Error"
+                                                }
+                                        }
+				}
+			},
+			"put": {
+				"summary": "Create or update a policy (sub)tree",
+				"tags": [ "policy" ],
+				"parameters": [
+					{ "$ref": "#/parameters/policy-path" },
+					{ "$ref": "#/parameters/policy-tree" }
+				],
+				"responses": {
+					"200": {
+                                                "description": "Success",
+                                                "schema": {
+							"$ref": "#/definitions/PolicyTree"
+                                                }
+					},
```

Comment:
Yes, I think this should include a `PolicyValidationError` which includes type of syntax error, line number, etc. Will add this.